demos/bombs/bombs.cpp

```diff
@@ -196,9 +197,11 @@ void BombsFrame::OnEasyCorner(wxCommandEvent& WXUNUSED(event))
                this
              );
 
-    if(ok!=wxYES)return;
+    if(ok!=wxYES)return GetMenuBar()->Check(bombsID_EASYCORNER, m_easyCorner);
```

Comment:
Minor, but we don't put compound statements on the same line. Also, a comment would probably be useful:
```suggestion
    if(ok!=wxYES)
    {
        // Undo the change to the menu bar, to keep it in sync with the actual option value.
        return GetMenuBar()->Check(bombsID_EASYCORNER, m_easyCorner);
    }
```